### DISCUSSION TITLE
Google reCaptcha field

### DISCUSSION THREAD

**Discussion by AbanoubNassem** [filamentphp/filament#4699] [Showcase]
I have made a [`google-recaptcha-field`](https://github.com/AbanoubNassem/filament-grecaptcha-field)  for `filament-forms` and uploaded it to the plugins section.

it's been like 4 days now, still pending approval , how long it takes to be approved? 

https://filamentphp.com/plugins/google-recaptcha-field

**Comment by danharrin** [comment]
I've just approved it

**Reply by AbanoubNassem** [reply]
thank you.
